sunpy/map/tests/test_mapbase.py

```diff
@@ -1181,6 +1181,27 @@ def test_quicklook(aia171_test_map):
         assert aia171_test_map._repr_html_() in html_string
 
 
+# requires dask array to run properly
+def test_dask_array(aia171_test_map):
+    dask_array = pytest.importorskip('dask.array')
```

Comment:
Do we need to have this? I feel like we run our test suite with all our optional dependencies installed.

Reply:
In setup.cfg, the dask dependency is under its own extras heading and is not included under the test heading so I would say this is the right approach. This is also how the tests are setup for dask arrays in the map factory so I think this is the best approach.